### PR TITLE
HW13 Tests for BusTickets

### PR DESCRIPTION
**Pull request by e-kondra**
1. Attach JUnit 5 and Mockito dependencies to your project (via Spring Boot starter or independently)
2. Pick the * biggest_service _class(C) in your application.
3. Create a Test class for * biggest_service _class(C) in src.test.java.com.andersen.$[YOUR_PROJECT_NAME}
4. Use this class to create @Test methods for all methods in * biggest_service _class(C)
5. Every method should be covered by at least 3 test methods - positive, negative and corner case.
6. Use JUnit annotations and method to “init” test data and make assertions and verification checks.
7. Use Mockito to mock DB repository behaviour.

### CODE REVIEW

File: src/test/java/andersen/AndersenTasks/service/BusTicketService_Test.java [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+@ExtendWith(MockitoExtension.class)
+class BusTicketService_Test {
+
+    @Mock
+    BusTicketRepository mockRepository;
+    @InjectMocks
+    BusTicketService_ service;
+
+    User user;
```
Comment (deolexx) [inline src/test/java/andersen/AndersenTasks/service/BusTicketService_Test.java:36]: fields should be private

**Review by deolexx — APPROVED** [review]
